src/utils/platform/f4xx/timef4xx.cpp

```diff
@@ -15,11 +15,11 @@ namespace EVT::core::time {
 */
 void wait(uint32_t ms) {
     // TODO: Remove need for this first time call
```

Comment:
Delete this or leave it, but don't comment it out. Also, if you're going to do this, do it in f3xx as well